homebridge/config.schema.json

```diff
@@ -8,13 +8,18 @@
       "email": {
         "title": "Email",
         "type": "string",
+        "required": true,
         "format": "email",
         "placeholder": "Email"
       },
       "password": {
         "title": "Password",
         "type": "string",
-        "placeholder": "Password"
+        "placeholder": "Password",
+        "required": true,
+        "options": {
+            "hidden": true
```

Comment:
I'm fine with accepting this `options` section, however I'm not sure how useful it is since you can click over to the "Config" tab and see everything in plain text anyways

Reply:
It is useful for HOOBS interface because we hide it in the UI unless you open the config file